routing_common/vehicle_model.hpp

```diff
@@ -67,7 +70,7 @@ class VehicleModel : public VehicleModelInterface
   {
     char const * m_types[2];  /// 2-arity road type
     double m_speedKMpH;       /// max allowed speed on this road type
-    bool m_isTransitAllowed;  /// transit allowed for this road type
+    bool m_isPassThroughAllowed;  /// pass through this road type is allowed
```

Comment:
Could you please align all comments to these fields?

Reply:
done